### PR TITLE
[WIP] Try CI with our images - 3

### PR DESCRIPTION
**Pull request by aliok**

### CODE REVIEW

File: hack/lib/strimzi.bash [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  oc -n kafka apply --selector strimzi.io/crd-install=true -f "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml"
+  curl -L "https://github.com/strimzi/strimzi-kafka-operator/releases/download/${strimzi_version}/strimzi-cluster-operator-${strimzi_version}.yaml" \
+  | sed 's/namespace: .*/namespace: kafka/' \
+  | oc -n kafka apply -f -
+
+  header "Applying Strimzi Cluster file"
+  oc -n kafka apply -f "https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/${strimzi_version}/examples/kafka/kafka-persistent.yaml"
+
+  header "Waiting for Strimzi to become ready"
+  kubectl wait deployment --all --timeout=-1s --for=condition=Available -n kafka
```
Comment (matzew) [inline hack/lib/strimzi.bash:16]: the `wait` is available on `oc` too, right? :-)

**Comment by aliok** [comment]
This is green, awesome!

Created https://github.com/openshift-knative/serverless-operator/pull/561 to create proper code. I was mostly trying stuff in this PR.

**Comment by aliok** [comment]
/test all

Just trying again to see if things are still green

**Pull request closed without merging**